frontend/src/styles/HetComponents/HetCardExportMenuItem.tsx

```diff
@@ -16,13 +18,19 @@ export function HetCardExportMenuItem({
   children,
   className = '',
   iconProps = {},
+  spanClassName,
+  iconClassName,
 }: HetCardExportMenuItemProps) {
   return (
     <MenuItem className={`pl-3 ${className}`} onClick={onClick}>
-      <ListItemIcon className='flex items-center px-2 py-1'>
+      <ListItemIcon
+        className={`${iconClassName} flex items-center px-2 py-1 w-full`}
+      >
         <Icon className='mx-1 w-8' {...iconProps} />
         {children && (
-          <span className='text-altBlack text-small'>{children}</span>
+          <span className={`${spanClassName} text-altBlack text-small`}>
```

Comment:
To future-proof, we should try to always add these class overrides last in the string template, since tailwind classes get applied left to right. this way in the future if anyone ever hard codes a duplicate class onto this component, the override incoming prop class would take precedence

Reply:
oh right! nice catch, thanks ben 🔎